### DISCUSSION TITLE
courses/computer-structure/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8_01/05_%EB%85%BC%EB%A6%AC%EC%97%B0%EC%82%B0

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#67] [Q&A]
# courses/computer-structure/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8_01/05_%EB%85%BC%EB%A6%AC%EC%97%B0%EC%82%B0

https://lms.vanillacoding.co/courses/computer-structure/%EC%9E%90%EB%B0%94%EC%8A%A4%ED%81%AC%EB%A6%BD%ED%8A%B8_01/05_%EB%85%BC%EB%A6%AC%EC%97%B0%EC%82%B0

**Comment by kam1989** [comment]
오늘 수업에 참석하지 못해 죄송합니다. 끝까지 잘 봤습니다. 구글로 영상 공유되면 다시한번 보겠습니다.
